### PR TITLE
Add basic admin docs

### PR DESCRIPTION
**Pull request by davisagli**

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    plans:
+        install:
+            slug: install
+            title: Install
+            tier: primary
+            steps:
+                1:
+                    flow: install_prod
+
+This sets us up to publish the ``install_prod`` flow to a Plan that will be the primary (default) Plan for the Product and available with /install in the URL (from the slug).
```
Comment (wlonk) [inline docs/tutorial.rst]:
```suggestion
This sets us up to publish the ``install_prod`` flow to a Plan that will be the primary (default) Plan for the Product and available with ``/install`` in the URL (from the slug).
```

File: docs/tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+4. Connect CumulusCI to MetaDeploy
+==================================
+
+This is a one-time setup task.
+In the MetaDeploy admin UI, go to the Tokens object (under Auth Tokens) and create one for your user.
```
Comment (wlonk) [inline docs/tutorial.rst]: You can also use the `manage` command, if you have access to that. Should that be included here, or does this presume only web access?

Comment (davisagli) [reply docs/tutorial.rst]: I'm fine mentioning it, but the web-based version is easier to explain given it's what we're describing above.

Comment (wlonk) [reply docs/tutorial.rst]: Then I have no attachment to mentioning it.

**Pull request merged**